Foodhut/src/styles/sections/header.css

```diff
@@ -1,81 +1,68 @@
 /* Style for desktop screen width = 1440px */
 .header-container {
-  display: flex;
-  justify-content: space-between;
   align-items: center;
+  gap: 16.625rem;
 }
 
 .wrapper-navbar-container {
-  display: flex;
-  gap: 70px;
+  gap: 4.375rem;
   align-items: center;
 }
 
 .navbar-container {
-  display: flex;
-  gap: 50px;
-  list-style-type: var(--hidden-display);
+  gap: 3.125rem;
+  list-style-type: none;
 }
 
 .navbar-container :hover {
   color: var(--color-primary);
 }
 
 .nav-item {
-  font-size: var(--font-size-large);
+  font-size: var(--font-size-normal);
   color: var(--color-dark);
-  text-decoration: var(--hidden-display);
+  text-decoration: none;
 }
 
 .navbar-button {
-  display: var(--hidden-display);
+  display: none;
 }
 
 /* Style for desktop screen under 1300px*/
 @media (max-width: 1300px) {
   .logo-company-container {
-    width: 97px;
-    height: 62px;
+    width: 6.0625rem;
+    height: 3.875rem;
   }
 
   .navbar-container {
-    gap: 20px;
+    gap: 1.25rem;
   }
 
   .nav-item,
   .button {
     font-size: var(--font-size-small);
   }
-
-  img {
-    width: 100%;
-    height: auto;
-  }
 }
 
 /* Style for desktop tablet and mobile screen */
 @media (max-width: 768px) {
   .logo-company-container {
-    width: 97px;
-    height: 62px;
+    width: 6.0625rem;
+    height: 3.875rem;
   }
-  
+
   .navbar-button {
     display: block;
-    border: var(--hidden-display);
+    border: none;
     background-color: transparent;
   }
 
   .navbar-button :hover {
     cursor: pointer;
   }
 
-  .wrapper-navbar-container {
-    display: var(--hidden-display);
-  }
-
-  img {
-    width: 100%;
-    height: auto;
+  .logo-company-container + .flex {
```

Comment:
Avoid to restyle common styles. Pls check all

Reply:
i fixed it, thanks for your contribution